client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -323,6 +323,8 @@
         hidden-input="r_text"
         v-model="values.text" />
 
+      <slot />
+
```

Comment:
why do You split the business-logic from the button? 
If there is a case, where we don't want to offer the quicksave, shouldnt we then use a flag or move the logic to the triggering component?

Reply:
That is a good question. Basically I thought we only need the quick save button after the ocr text recognition step right before confirming the pdf import. But it makes me wonder now if this is a useful feature for direct imports in the import center. 

I mean I have added the slot so it does not interfere with the import center but if we want to quick save the statement text there too, I think you are right by moving the button directly into DpSimplifiedNewStatementForm.vue but here I am not sure about the user workflow. I always thought they use the direct import only with copy+paste but if this is not the case I agree and we should move the button to the form directly.

Reply:
No. I think You are right, a  we don't need it in the import center. But then I would suggest to move the methods to the dpConvertAnotatedPdf file, to gather button-template and method in the same file.

Reply:
Thanks for the suggestion. Moved the function to `DpConvertAnnotatedPdf.vue` here: https://github.com/demos-europe/demosplan-core/pull/3992/commits/411080c03c604beb58c87e75ed090ecd12c0b5ae but I think we still need the ref usage since the v-model of `values.text` contains the current text of the statement text editor (which exists in `DpSimplifiedNewStatementForm.vue`).